doc/sphinx/install/ubuntu.md

```diff
@@ -0,0 +1,96 @@
+(sec-install-ubuntu)=
+# Ubuntu Packages
+
+As of Cantera 3.0.0, packages are available for Ubuntu 20.04 (Focal Fossa), Ubuntu 22.04
+(Jammy Jellyfish), Ubuntu 23.04 (Lunar Lobster), and Ubuntu 23.10 (Mantic Minotaur). To
+see which Ubuntu releases and Cantera versions are currently supported, visit the
+[Cantera PPA](https://launchpad.net/~cantera-team/+archive/ubuntu/cantera).
```

Comment:
I wonder if it's worth adding something like, non-LTS versions are supported on a best-effort basis and in general for X timeframe?